pymc/tests/distributions/test_continuous.py

```diff
@@ -778,10 +778,13 @@ def test_moyal_logp(self):
         reason="PyMC underflows earlier than scipy on float32",
     )
     def test_moyal_logcdf(self):
+        # SciPy as precision issues at {mu=-2.1, sigma=0.5, x=2.1}
```

Comment:
```suggestion
        # SciPy has new (?) precision issues at {mu=-2.1, sigma=0.5, x=2.1}
```

Reply:
it might also be that PyTensor now has smarter optimizations and therefore no longer runs into the same precision issue?

Our logcdf was [deterministic at -2227.95939](https://github.com/pymc-devs/pymc/actions/runs/3665014447/jobs/6195811961#step:7:780):

```
FAILED pymc/tests/distributions/test_continuous.py::TestMatchesScipy::test_moyal_logcdf - AssertionError: 
Arrays are not almost equal to 6 decimals
{'mu': array(2.1), 'sigma': array(0.5), 'value': array(-2.1)}
x and y -inf location mismatch:
 x: array(-2227.95939)
 y: array(-inf)
```

This value is plausible; at `sigma=0.9` it's around -50.

Reply:
I don't think we did any stabilization improvements recently. Regardless, part of my suggestion was to also fix the typo in the original comment (as instead of has)